templates/bundles/DemosPlanCoreBundle/DemosPlanMap/map_public_participation_detail.html.twig

```diff
@@ -1,369 +1,371 @@
-    {#                    data-toggle-exclusive>#}
-                 --><li
-                        class="{{ 'layout__item u-1-of-3'|prefixClass }}"
-    {#                    data-toggle="#queryArea"#}
-                        :class="activeActionBoxTab === 'priorityArea' ? prefixClass('is-visible-actionbox-toggle') : ''"
-                        @click.prevent="update({ key: 'activeActionBoxTab', val: 'priorityArea' })">
-                        <a
-                            class="{{ 'c-actionbox__toggle'|prefixClass }}"
-                            data-cy="mapPublicParticipationDetail:queryAreaButton"
-                            href="#queryAreaButton"
-                            aria-label="{{ 'statement.map.choose_priority_area'|trans}}">
-                            <i class="{{ 'fa fa-flag-o'|prefixClass }}" aria-hidden="true"></i></a>
-                    </li><!--
-                    {% endif %}
-
-    {#             --><li#}
-    {#                    class="{{ 'layout__item js__toggleAnything'|prefixClass }} {{ procedureStatementPriorityArea ? 'u-1-of-3'|prefixClass : 'u-1-of-2'|prefixClass }}"#}
-    {#                    data-toggle="#{{ hasPermission( 'feature_map_use_drawing_tools' ) ? 'drawTools' : 'markLocation' }}"#}
-    {#                    data-toggle-container="{{ '.c-actionbox'|prefixClass }}"#}
-    {#                    data-toggle-property="{{ '.is-visible-actionbox'|prefixClass }}"#}
-    {#                    data-toggle-exclusive>#}
-                 --><li
-                        v-show="{{ mapAndCountyGroupEnabled }}"
-                        class="{{ 'layout__item'|prefixClass }} {{ procedureStatementPriorityArea ? 'u-1-of-3'|prefixClass : 'u-1-of-2'|prefixClass }}"
-                        :class="activeActionBoxTab === 'draw' ? prefixClass('is-visible-actionbox-toggle') : ''"
-                        @click.prevent="update({ key: 'activeActionBoxTab', val: 'draw' })">
-                        <a
-                            class="{{ 'c-actionbox__toggle'|prefixClass }}"
-                            data-cy="mapPublicParticipationDetail:markLocationButton"
-                            href="#markLocationButton"
-                            aria-label="{{ "statement.map.draw_to_map"|trans }}" aria-describedby="statementMapDrawHint">
-                            <i class="{{ 'fa fa-map-marker'|prefixClass }}" aria-hidden="true"></i>
-                        </a>
-                    </li>
-                </ul>
-
-                <div class="{{ 'c-actionbox__panelwrapper u-pb-0 u-mb-0_5'|prefixClass }}">
-                    <div v-show="activeActionBoxTab === 'talk'" id="statementAction" class="{{ 'c-actionbox__panel'|prefixClass }}"
-                         :class="activeActionBoxTab === 'talk' ? prefixClass('is-visible-actionbox') : ''"
-                         data-statement-state-started>
+<div class="c-map__toolbar">
+    <div class="{{ 'c-map__toolbar--container u-mb-0_5-palm'|prefixClass }}">
```

Comment:
The `--` separator denotes a [modifier in BEM](https://getbem.com/naming/#modifier). As the newly introduced element does not implement a variation of the toolbar, but introduces a new "block", the appropriate naming would be `c-map__toolbar-container`. Also see our [conventions on selector naming](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/coding-styleguides/css.html#selector-naming).

On the other hand, this whole thing could possibly be expressed using Tailwind with `sm:overflow-y-auto sm:height-[531px] sm:pr-1`. I know, the `sm:` breakpoint kicks in 1px earlier than the `media-query('lap-up')` include but maybe we can ignore that.

Reply:
right, thx for the information. I refactored it here

Regarding the Tailwind, I am not really sure why, but it did not work with prefixes like `sm:` there